docker-compose.yml

```diff
@@ -564,14 +612,14 @@ services:
       - TIMEOUT_LENGTH=120
     command: aerospike:3000 servicestackredis:6379 stackexchangeredis:6379 elasticsearch5:9200 elasticsearch6:9200 elasticsearch7:9200 sqlserver:1433 mongo:27017 postgres:5432 mysql:3306 mysql57:3306 rabbitmq:5672 kafka-broker:9092 kafka-zookeeper:2181 aws_sqs:9324 couchbase:11210
 
-  IntegrationTests.ARM64:
+  IntegrationTests.ARM64.Debugger:
```

Comment:
It _looks_ like you've replaced the `IntegrationTests.ARM64` target here, rather than added a new one?

Reply:
It was already fixed  😅